natgeo/mortar
# [AEM-6339] Fixes bug where kicker doesn't render if it's empty in AEM

https://jira.natgeo.com/browse/AEM-6399

To test:

- Pull AEM-6399
- Run `npm test` to verify tests/linting
- Run `gulp serve`, verify component renders
- Run `gulp build`
- From within the mortar directory, `cp lib/modules/podpromo/PodPromo.js ~/Documents/workspace/ngs/aem-all/aem-platform/platform-view/node_modules/@natgeo/mortar_v3/lib/modules/podpromo/.` (**note second path should match wherever you have your AEM code)**
- From within the mortar directory, `cp lib/modules/podpromo/_styles.scss ~/Documents/workspace/ngs/aem-all/aem-platform/platform-view/node_modules/@natgeo/mortar_v3/lib/modules/podpromo/.` (**note second path should match wherever you have your AEM code)**
- From platform-view in AEM, run `mvn clean install -Pauto-deploy -s ../settings.xml`
- Create POD component with a kicker, verify it renders.
- Remove the kicker and verify the POD component still renders
- Hit the merge button
